server/liveblog/blogposting_schema/utils.py

```diff
@@ -176,13 +201,18 @@ def generate_schema_for(blog, posts, theme_settings={}):
     if not posts:
         return ""
 
+    tz_offset = theme_settings.get("jsonldTimeZone", 0)
+    start_date = convert_to_configured_timezone(blog.get("start_date"), tz_offset)
+    modified_date = convert_to_configured_timezone(get_modified_date(blog), tz_offset)
+    end_date = convert_to_configured_timezone(blog.get("end_date", None), tz_offset)
+
     liveblogposting = LiveBlogPosting(
         headline=blog["title"],
         description=blog["description"],
-        date_published=blog["start_date"],
-        date_modified=get_modified_date(blog),
-        coverage_start_time=blog["start_date"],
-        coverage_end_time=blog.get("end_date", None),
+        date_published=start_date,
+        date_modified=modified_date,
+        coverage_start_time=start_date,
+        coverage_end_time=end_date,
```

Comment:
let's also add this improvement to the posts 
<img width="1000" height="783" alt="image" src="https://github.com/user-attachments/assets/5b0bdffb-dd15-41c3-b252-e200a521adec" />